.github/workflows/maven.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         # os: [macos-latest, ubuntu-latest, windows-latest]
         os: [ubuntu-latest]
-        java-version: [16]
+        java-version: [17.0.0-ea.35]
```

Comment:
Why is this such a specific version and not just 17?

Reply:
Hey @donraab, I actually tried 17 first and received the following error: https://github.com/rinatigati/CodeKatas/runs/3404410626?check_suite_focus=true

You might need to expand the Setup JDK stage to view it. In short, that version could not be found. I'm thinking it's because 17 is still in early access so because it hasn't officially been released that doesn't map to it yet. I'm thinking after it's out, we should be able to update it to 17 and it should work then.

For now I specified the latest early access build of 17.

Reply:
Made changes to add a separate step to download JDK and then use that in the setup JDK step. Doesn't use a specific version anymore and downloads the JDK from jdk.java.net.